### DISCUSSION TITLE
Ability to adjust Gemini safety settings

### DISCUSSION THREAD

**Discussion by zavocc** [Taewan-P/gpt_mobile#55] [Ideas]
Has such option in AI studio and the API, it would be extremely helpful for sensitive chat tasks that may involve permissive safety triggers and sometimes some innocent prompts are mistakenly triggers safety filter which can be ridiculous to deal with

**Comment by Amin2460** [comment]
gemini api also has "grounding"(web search) functionality for the api models, it'd be awesome if you could support that too 🙏 

that and supporting multimodal models. for example gemini models can take text,image,video,audio as input, and they can output text,image,audio. it'd be amazing if this app could support that one day.

btw, thank you for making this app ❤️
